tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -385,7 +385,7 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    '          "--transform LowLatency2[use_const_initializer=False]" or ' +
                                    '          "--transform \"MakeStateful[param_res_names='
                                    '{\'input_name_1\':\'output_name_1\',\'input_name_2\':\'output_name_2\'}]\"" ' +
-                                   'Available transformations: "LowLatency2", "MakeStateful"',
+                                   'Available transformations: "LowLatency2", "MakeStateful", "Pruning", "ConvertSequenceToTensorIterator"',
```

Comment:
@bszmelcz, please explain the reason to have it in MO. Now the POT can import this transformation internally and this is sufficient.
I propose to remove it because now there is no need in it.

Reply:
That makes sense, I will remove it then